tests/model_fields/models.py

```diff
@@ -187,6 +187,14 @@ class GenericIPAddress(models.Model):
     ip = models.GenericIPAddressField(null=True, protocol='ipv4')
 
 
+class DecimalDigitsEqualDecimalPlaces(models.Model):
```

Comment:
It's better to reuse existing models if possible since adding more models makes the test suite slower and more complex. You could delete the `DecimalLessThanOne` model in favor of this one.